### PR TITLE
ctp: AttestationStation deterministic proxy

### PR DESCRIPTION
**Pull request by tynes**
**Description**

Deploys the proxy in a deterministic manner for
the attestationstation

### CODE REVIEW

File: packages/contracts-periphery/deploy/op-nft/AttestationStationProxy.ts [hunk 1]
```diff
@@ -16,22 +13,33 @@ const { getAddress } = utils
  */
 const deployFn: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
   const { deployer } = await hre.getNamedAccounts()
+  const ddd = hre.deployConfig.ddd
+
+  if (getAddress(deployer) !== getAddress(ddd)) {
+    throw new Error('Must deploy with the ddd')
+  }
```
Comment (tynes) [inline packages/contracts-periphery/deploy/op-nft/AttestationStationProxy.ts:20]: This assertion is here to ensure that the impl and admin can be set with deterministic deployment

File: packages/contracts-periphery/config/deploy/hardhat.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { DeployConfig } from '../../src'
 
 const config: DeployConfig = {
-  ddd: '0x9C6373dE60c2D3297b18A8f964618ac46E011B58',
+  ddd: '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266',
   l2ProxyOwnerAddress: '0x70997970c51812dc3a010c7d01b50e0d17dc79c8',
```
[the remark this answers is not in the document]
Comment (smartcontracts) [reply packages/contracts-periphery/config/deploy/hardhat.ts:5]: It needs to be the same address if you want the local deployment of the proxy to be deployed at the same address as every other network

File: packages/contracts-periphery/deploy/op-nft/AttestationStationProxy.ts (hunk 1 above)
Comment (smartcontracts) [reply packages/contracts-periphery/deploy/op-nft/AttestationStationProxy.ts:20]: The `ddd` doesn't need to be the deployer. It just needs to be the owner.

Comment (tynes) [reply packages/contracts-periphery/deploy/op-nft/AttestationStationProxy.ts:20]: I was going for a single key deploy + config. It feels like more work to deploy with one key then use another key to config

**Review by smartcontracts — APPROVED** [review]

File: packages/contracts-periphery/config/deploy/hardhat.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { DeployConfig } from '../../src'
 
 const config: DeployConfig = {
-  ddd: '0x9C6373dE60c2D3297b18A8f964618ac46E011B58',
+  ddd: '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266',
```
[the remark this answers is not in the document]
Comment (tynes) [reply packages/contracts-periphery/config/deploy/hardhat.ts:4]: this is the first account in the hh keys, `test test test... junk` i think `m/44'/60'/0/1`

File: packages/contracts-periphery/config/deploy/hardhat.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (tynes) [reply packages/contracts-periphery/config/deploy/hardhat.ts:5]: i changed this so that we could deploy to a local hh network with no issues

**Pull request merged**